style/modules/header.css

```diff
@@ -0,0 +1,47 @@
+.header {
+    min-width: 500px;
+    display: block;
+    display: flex;
+    align-items: center;
+}
+
+.header > *:not(.header--linkLogo) {
```

Comment:
I've been playing around with trying to get it more like Figma, and this is what I think we should do:

- move `*:not(.header--linkLogo)` rules into `.header--link`
- change `.header--link` padding to 4px
- add a `.header--link-tight` variant
- wrap the last two with `<a class="header--link header--link-tight">`
- change `flex-grow` values to:
  - header--linkLogo: 8
  - header--link: 1
  - header--link-tight: 0.5

This way:

- there's some slight minimum spacing (the 4px padding)
- the spacing grows (the flex-grow)
- the spacing grows only a little, so that the logo stays set apart (the relatively high flex-grow on the logo)
- the buttons are tighter, because they have colored backgrounds which spaces the text out already (the `header--link-tight` variant)


All in all, it should look like this:

```css
.header--linkLogo {
    flex-grow: 8;
    margin-right: auto;
}

.header--link {
    padding: 4px;
    flex-grow: 1;
    text-align: center;
    text-decoration: none;
    font-size: 16px;
    color: var(--gray);
}

.header--link-tight {
    flex-grow: 0.5;
}
```

and 

```html
<a class="header--link header--link-tight" href="#">
  <span class="btnLink btnLink-small">Get Updates</span>
</a>
<a class="header--link header--link-tight" href="#">
  <span class="btnLink btnLink-red btnLink-small">Donate</span>
</a>
```

Reply:
this is how it should look with those changes:

narrow

<img width="979" alt="narrow" src="https://user-images.githubusercontent.com/733916/97049931-5cd55e00-1531-11eb-8fef-43e3b5029a97.png">

wide
<img width="1648" alt="wide" src="https://user-images.githubusercontent.com/733916/97049933-5d6df480-1531-11eb-98f7-66fc1dd3140a.png">